workflows/virology/influenza-isolate-subtyping/influenza-isolate-subtyping.ga

```diff
@@ -0,0 +1,2156 @@
+            "tool_version": "2.3.3+galaxy0",
+            "type": "tool",
+            "uuid": "a4745843-80b5-4017-94f3-4ed895e3d08c",
+            "when": "$(inputs.when)",
+            "workflow_outputs": [
+                {
+                    "label": "iq-tree_mldist",
+                    "output_name": "mldist",
+                    "uuid": "c5d752a6-d45f-46ad-a7fd-600b28ad5454"
+                },
+                {
+                    "label": "iqtree_report",
+                    "output_name": "iqtree",
+                    "uuid": "b8fc5bb9-c8dc-4db1-be8c-69de315bff49"
+                },
+                {
+                    "label": "iq-tree_treefile",
+                    "output_name": "treefile",
+                    "uuid": "08ab78b7-2612-4315-ab3a-fb2588c82172"
```

Comment:
These three WF outputs need to be given a human-readable name as a post-job action.
Sth like IQ-Tree per-segment ML distance matrix, IQ-Tree per-segment ML tree and IQ-Tree per-segment report.